snapcraft.yaml

```diff
@@ -1572,6 +1573,13 @@ parts:
           strip -s ${CRAFT_PRIME}/${zfs}/bin/* ${CRAFT_PRIME}/${zfs}/lib/*
       done
 
+      if [ "$(uname -m)" != "riscv64" ]; then
+          # Prime the documentation only if the arch is not riscv64.
+          # Some python dependencies are not available for riscv64 or just require a build from source.
+          # Not worth the effort for now.
+          cp -r ${CRAFT_STAGE}/share/lxd-documentation ${CRAFT_PRIME}/share/
```

Comment:
Nearly there. :) You should really be using `""` to wrap your variables to prevent potential command injection.
You've done it for the `cp` and `mkdir` calls above, but haven't maintained that detail here.